src/autonlp/evaluate.py

```diff
@@ -43,3 +45,52 @@ def __str__(self):
             ]
         )
         return output
+
+
+DATASETS_TASKS = ["text_classification"]
+AUTONLP_TO_DATASETS_TASKS = {
+    "text_classification": "text-classification",
+    "extractive_question_answering": "question-answering-extractive",
+}
+DATASETS_TO_AUTONLP_TASKS = {v: k for k, v in AUTONLP_TO_DATASETS_TASKS.items()}
```

Comment:
perhaps these three variables belong in `autonlp.tasks`?

Reply:
ummm `text_classification` is not a valid AutoNLP task.

Reply:
yes you are correct. here i am just using `DATASETS_TO_AUTONLP_TASKS` to inform the user which values of `task` can be provided in the CLI if no compatible task templates are found: 

https://github.com/huggingface/autonlp/pull/81/files#diff-91508abf082abfe6948618a97ff988de77365bb0400235c6da96e9510c758234R88

i can remove this mapping and simplify the error message if that would help